cluster/mesos/cluster.go

```diff
@@ -72,10 +73,15 @@ func NewCluster(scheduler *scheduler.Scheduler, store *state.Store, TLSConfig *t
 	// Empty string is accepted by the scheduler.
 	user, _ := options.String("mesos.user", "SWARM_MESOS_USER")
 
+	hostname, _ := os.Hostname()
+	if hostname == "" {
+		hostname = frameworkName
```

Comment:
the official image we produce has no hostname. we need to give something otherwise mesos-go fails